.github/workflows/test.yml

```diff
@@ -0,0 +1,145 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  #### TEST STAGE ####
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      # Keys:
+      # - coverage: Whether to run the tests with code coverage.
+      # - experimental: Whether the build is "allowed to fail".
+      matrix:
+        php: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4']
+        coverage: [false]
+        experimental: [false]
+
+        include:
+          # Run code coverage on high/low PHP.
+          # Note: use PHP 7.4 as "high" as PHPUnit 7 in combination with PHP 8 will fail to run coverage (NEEDS CHECKING)
```

Comment:
The setup seems to use 8.0 now. Does this mean this was checked and the note is out-of-date?

Reply:
Correct, removing it now.